nannou/src/draw/renderer/shaders/shader.frag

```diff
@@ -18,19 +18,18 @@ layout(location = 2) flat in uint v_mode;
 layout(location = 0) out vec4 f_color;
 
 void main() {
+    vec4 f_color_texture = texture(sampler2D(tex, tex_sampler), v_tex_coords);
+    float tex_a = texture(sampler2D(text, text_sampler), v_tex_coords).r;
```

Comment:
Maybe these two variable names could be a bit clearer now that they're split out? i.e. something like

```glsl
    vec4 tex_color = texture(sampler2D(tex, tex_sampler), v_tex_coords);
    float text_alpha = texture(sampler2D(text, text_sampler), v_tex_coords).r;
```